src/com/tw/go/plugin/jgit/JGitHelper.java

```diff
@@ -293,6 +293,11 @@ public void fetch(String refSpec) {
     }
 
     @Override
+    public void fetch(String refSpec, Integer depth) {
+        fetch(refSpec); //Does not honour depth
```

Comment:
Sure, I'll change this -- I've never done this in Java so let me know if what I do is wrong.